react/components/RequestOrganizationForm.tsx

```diff
@@ -570,9 +615,15 @@ const RequestOrganizationForm: FC = () => {
                     !isValidAddress(addressState) ||
                     (formState.phoneNumber &&
                       !validatePhoneNumber(formState.phoneNumber)) ||
-                    customFieldsState?.some((customField: CustomField) => {
+                    // custom fields need to be filled
```

Comment:
Are we making all custom fields required on signup? Not sure if that's what we want...

Reply:
Currently only 1 field in the forms is not required. Can we keep them all required for now? In case there's a requirement to make required/not required per field, I can add it afterwards.